### DISCUSSION TITLE
How to write a script to query the comments like """comment""" for python

### DISCUSSION THREAD

**Discussion by BaiChienKao** [github/codeql#12219] [Q&A]
I would like to write a ql script to query the comments in python. I tried the Comment as below
```ql
import python
from Comment com
select com
```
But it cannot query the unassigned-string type comments like:
```py
class TemplateBuilder(Builder):
    """Load template file.
    Loads the content of the template_file and the authz_def (section of template file with fixed authorization
    definitions) into two key-value pairs in the build context.
    """
    def initialize(self, ctx): .........
```
I was wondering how can I have a script to find this kind of comments.
Thank you so much.

**Comment by tausbn** [comment]
You'll want the `getDocString` method on the `Scope` class. Something like
```ql
select any(Scope s).getDocString()
```
should give you all the docstring comments (as `StrConst`s).
